packages/core/src/icon/icon.stories.ts

```diff
@@ -74,138 +47,193 @@ export default {
   },
 };
 
-export function all() {
-  const search = text('search', '', propertiesGroup);
-  const size = select(
-    'size',
-    { xs: 'xs', 'sm (default)': 'sm', md: 'md', lg: 'lg', xl: 'xl', xxl: 'xxl' },
-    'lg',
-    propertiesGroup
-  );
-  const dir = select(
-    'direction',
-    { 'up (default)': undefined, down: 'down', left: 'left', right: 'right' },
-    undefined,
-    propertiesGroup
-  );
-  const fl = select(
-    'flip',
-    { 'none (default)': undefined, vertical: 'vertical', horizontal: 'horizontal' },
-    undefined,
-    propertiesGroup
-  );
-  const badge = select(
-    'badge',
-    {
-      'none (default)': '',
-      info: 'info',
-      success: 'success',
-      warning: 'warning',
-      danger: 'danger',
-      inherit: 'inherit',
-      'warning-triangle': 'warning-triangle',
-      'inherit-triangle': 'inherit-triangle',
-    },
-    '',
-    propertiesGroup
-  );
-  const iconStatus = select(
-    'status',
-    { 'none (default)': '', info: 'info', success: 'success', warning: 'warning', danger: 'danger' },
-    '',
-    propertiesGroup
-  );
-  const inverse = boolean('inverse', false, propertiesGroup);
-  const solid = boolean('solid', false, propertiesGroup);
-
-  function createIconIndices(icons: IconShapeTuple[], aliases: [string, string[]][]): string[] {
-    const iconsMap = icons.map(iconTuple => iconTuple[0]);
-    const aliasMap = aliases.map(aliasTuple => aliasTuple[1]).reduce((acc, val) => acc.concat(val), []);
-    return iconsMap.concat(aliasMap).sort();
+@customElement('demo-all-icons')
```

Comment:
Still working on this. Likely a followup PR. Actions/Knobs don't quite work in storybook. This demo I replaced the knobs with a simple list of icons since we have the full demo on the new website now.